.changeset/slow-tigers-fail.md

```diff
@@ -0,0 +1,18 @@
+---
+'@pothos/plugin-errors': patch
```

Comment:
This is probably one on the most detailed and useful changesets to date!

This change should probably be a minor though, as it's adding a new feature